frontend/src/constants/routes.ts

```diff
@@ -1,6 +1,5 @@
-// TODO: Update landing route to correct route.
 export const LANDING_ROUTE = '/'
-export const ROOT_ROUTE = '/'
```

Comment:
This change breaks the bookmark routing I have added, since the hashes will never match the `/workspace` prefix 🤔